Master/healthcheck.js

```diff
@@ -0,0 +1,111 @@
+/* ----------------------------------------------------------------------------
+ * @copyright (C) 2019, Alert Logic, Inc
+ * @doc
+ * 
+ * Various Event hub collector health checks.
+ * The last error code is EHUB000006
+ * 
+ * @end
+ * ----------------------------------------------------------------------------
+ */
+ 
+const async = require('async');
+const azure = require('azure');
+const parse = require('parse-key-value');
+
+function initArmEhub(master) {
+    const azureCreds = master.getApplicationTokenCredentials();
+    const subscriptionId = master.getConfigAttrs().subscription_id;
+    return azure.createEventHubManagementClient(azureCreds, subscriptionId);
+}
+
+function formatSdkError(master, alErrorCode, err) {
+    if (typeof err === 'string' || err instanceof String) {
```

Comment:
Is the check for 'string' necessary? Seems to be redundant with the 2nd check, no?

Reply:
if an `err` is contructed with `new String('some-error')` then `typeof err` returns `object`